### ISSUE TITLE
comet.params file as input

### ISSUE THREAD

**Issue by nahmedraja** [Workflomics/domain-annotations#1]
Based on my experience of running the workflow, I suugest to make comet.params as an input file and update Comet annotations accordingly.

**Comment by vedran-kasalica** [comment]
The problem is that annotating such input would be really difficult, as EDAM does not provide correct terminology (for example specifying it as a plain text file, would result in APE using other tools that can process text files, which should not be the case). Is providing the params file with the Comet.cwl a bit issue?

**Comment by nahmedraja** [comment]
No. it is not an issue. The user then have to manually copy the comet.prams file in the tools folder as currently the path to the comet.params file is statically set to "/cwl/tools/Comet/comet.params".

**Comment by vedran-kasalica** [comment]
In that case, I would keep it as it is, not to break something before leaving. Regarding the `comet.params` file, can we have a default params version and distribute it with Comet.cwl, so that we can run the pipeline without too many manual steps? Users can then change the file when needed.
